Storage/Storage/Model/Customer+CoreDataProperties.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+import CoreData
+
+
+extension Customer {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<Customer> {
+        return NSFetchRequest<Customer>(entityName: "Customer")
+    }
+
+    @NSManaged public var customerID: Int64
+    @NSManaged public var email: String?
+    @NSManaged public var firstName: String?
+    @NSManaged public var lastName: String?
+    @NSManaged public var billingFirstName: String?
+    @NSManaged public var billingLastName: String?
+    @NSManaged public var billingCompany: String?
+    @NSManaged public var billingAddress1: String?
+    @NSManaged public var billingAddress2: String?
+    @NSManaged public var billingCity: String?
+    @NSManaged public var billingState: String?
+    @NSManaged public var billingPostcode: String?
+    @NSManaged public var billingCountry: String?
+    @NSManaged public var billingPhone: String?
+    @NSManaged public var billingEmail: String?
+    @NSManaged public var shippingFirstName: String?
+    @NSManaged public var shippingLastName: String?
+    @NSManaged public var shippingCompany: String?
+    @NSManaged public var shippingAddress1: String?
+    @NSManaged public var shippingAddress2: String?
+    @NSManaged public var shippingCity: String?
+    @NSManaged public var shippingState: String?
+    @NSManaged public var shippingPostcode: String?
+    @NSManaged public var shippingCountry: String?
+    @NSManaged public var shippingPhone: String?
+    @NSManaged public var shippingEmail: String?
+    @NSManaged public var id: NSSet?
```

Comment:
This should be a descriptive name for the collection, and you can also add to the generated code to make it more helpful, e.g. 
```suggestion
    @NSManaged public var searchResults: Set<CustomerSearchResult>?
```

It's best to change it by first updating the Model, then regenerating the code, then editing the generated code.

E.g. [Product+CoreDataProperties](https://github.com/woocommerce/woocommerce-ios/blob/de959a276be9c197d97855c482ab242a573bda9e/Storage/Storage/Model/Product%2BCoreDataProperties.swift#L73)

Reply:
Ah yes! I remember we changed those in our session and I totally forgot. Changed, regenerated code, and edited  on da814bf07cd4f2c97e6fe2a9a325ffa15585cbd6